Logger.cs

```diff
@@ -9,33 +9,27 @@ public static class Logger
         private static string LogFilePath    => $"{Core.ModDirectory}/../cleaned_output_log.txt";
         private static string ModLogFilePath => $"{Core.ModDirectory}/{Core.ModName}.log";
 
+        public static StreamWriter writer;
```

Comment:
Changes to this file look perfect. Haven't had a chance to loook through others.